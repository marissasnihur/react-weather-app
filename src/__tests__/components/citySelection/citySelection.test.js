import React from 'react';
import ReactDOM from 'react-dom';

import CitySelection from '../../../components/CitySelection/CitySelection';

describe('CurrentCondition component', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CitySelection />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should match snapshot', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CitySelection/>, div);
    expect(div).toMatchSnapshot();
    ReactDOM.unmountComponentAtNode(div);
  });
});
