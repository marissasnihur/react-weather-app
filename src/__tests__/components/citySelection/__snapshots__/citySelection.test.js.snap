// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentCondition component should match snapshot 1`] = `
<div>
  <header>
    <label
      for="citySelection"
    >
      Select a city:
    </label>
    <div
      class="selectDropdown"
    >
      <select
        id="citySelection"
      >
        <option
          value="edmonton"
        >
          Edmonton
        </option>
        <option
          value="vancouver"
        >
          Vancouver
        </option>
        <option
          value="calgary"
        >
          Calgary
        </option>
        <option
          value="montreal"
        >
          Montreal
        </option>
        <option
          value="toronto"
        >
          Toronto
        </option>
      </select>
    </div>
  </header>
</div>
`;
